packages/core/src/resolvers/uriresolver.ts

```diff
@@ -1,13 +1,13 @@
+import isUrl from "is-url";
 import { Context } from "..";
 import { SubResolver } from "./subresolver";
 
 export function UriResolver(): SubResolver {
   // TODO: make a resolver that actually checks if something is a valid http link
```

Comment:
I'd say this comment is now deprecated, can be removed to not confuse developers